assets/files.wxs

```diff
@@ -1340,9 +1340,6 @@
         <Component Id="cmpB2DC1821DBE648078AEA75AC0AB62408" Guid="{4242bbb5-81dd-481d-a46a-6397d5bce4a5}">
```

Comment:
@SteveL-MSFT, your last commit had 4 failures in `PowerShell-CI-linux`
Clipboard cmdlet tests.Text.Get-Clipboard returns what is in Set-Clipboard
```powershell
Operation is not supported on this platform.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Clipboard.Tests.ps1: line 21
21:             Set-Clipboard -Value $guid

```
Clipboard cmdlet tests.Text.Get-Clipboard returns an array
```powershell
Operation is not supported on this platform.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Clipboard.Tests.ps1: line 28
28:             1,2 | Set-Clipboard

```
Clipboard cmdlet tests.Text.Get-Clipboard -Raw returns one item
```powershell
Operation is not supported on this platform.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Clipboard.Tests.ps1: line 36
36:             1,2 | Set-Clipboard

```
Clipboard cmdlet tests.Text.Set-Clipboard -Append will add text
```powershell
Operation is not supported on this platform.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Clipboard.Tests.ps1: line 42
42:             'hello' | Set-Clipboard

```